app/build.gradle

```diff
@@ -172,6 +172,8 @@ dependencies {
     implementation 'com.android.support:customtabs:28.0.0'
     implementation 'com.jaredrummler:android-device-names:2.0.0'
     implementation 'com.jakewharton.threetenabp:threetenabp:1.2.4'
+    implementation 'com.squareup.retrofit2:retrofit:2.9.0'
```

Comment:
Retrofit pls nepřidávej, teď ho na nic nepotřebuješ. Možná získání eHridu ani nebude HTTP request, ještě to není dořešené. Na tobě jenom tvorba UI s mock daty, networking nech kompletně na mě. Můžeš případně dávat TODO do kódu kde se má dělat volání na backend, aby se mi snadněji napojovalo.

Reply:
Retrofit odebrán, doplněno TODOčko do místa volání eHRID